packages/sourcecred/src/util/batch.test.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import {batchArray, batchIterator} from "./batch";
+
+describe("src/util/batch", () => {
+  describe("batch", () => {
+    it("batches an array bigger than the batch size", () => {
```

Comment:
Suggestion: add tests for corner cases of `batchSize`.

Reply:
accepted